bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/AbstractBigtableTable.java

```diff
@@ -276,25 +276,22 @@ public Result get(Get get) throws IOException {
   @Override
   public ResultScanner getScanner(final Scan scan) throws IOException {
     LOG.trace("getScanner(Scan)");
-    Span span = TRACER.spanBuilder("BigtableTable.scan").startSpan();
+    Span span = TRACER.spanBuilder("BigtableTable.readRows").startSpan();
```

Comment:
why is this changing from `scan` to `readRows`?

Reply:
Originally we traced the scope of  readRows() and iterator over rows, the scope is limited only to readRows() method.